### PR TITLE
Syn 203 change panel for selecting postprocessing methods

### PR DESCRIPTION
**Pull request by leny-mi**
## Describe your changes

- Fixes the slider spasms
- Adds advanced settings to control weights

## Checklist before requesting a review
- [x] I have performed a self-review of my code

### CODE REVIEW

File: packages/synthchron-tauri/src/components/editor/bottomDrawer/postprocessingPanel/PostprocessingPanel.tsx [hunk 1]
```diff
@@ -1,76 +1,78 @@
...
   PostprocessingStepType,
 } from '@synthchron/types'
 
-import { SortableItem } from './SortableItem'
+import { PostprocessingCheckboxLine } from './PostprocessingCheckboxLine'
 
 const postprocessingText = `Apply postprocessing steps to the generated traces to add noise. 
-The weight chooses which postprocessing step should be applied. 
-Currently, only one type of noise can be applied to each event.`
+The noise probability controls how whether noise should be applied to an event.
```
Comment (MathiasBoi6) [inline packages/synthchron-tauri/src/components/editor/bottomDrawer/postprocessingPanel/PostprocessingPanel.tsx:25]: 'how whether' is bad
I suggest -> The noise probability controls how much noise should be applied to an event.
or -> The noise probability controls whether noise should be applied to an event.

File: packages/synthchron-tauri/src/components/editor/bottomDrawer/postprocessingPanel/PostprocessingPanel.tsx [hunk 2]
```diff
@@ -83,6 +85,41 @@ const PostprocessingPanel: React.FC<PostprocessingPanelProps> = ({
...
+      }))
+    }, 75),
+    [setPostprocessing]
+  )
+
+  const updateWeight = useCallback(
+    (index: number, value: number) => {
+      const oldSum = weightValues.reduce((a, b) => a + b, 0)
+      const oldFactionOfOthers = (oldSum - weightValues[index]) / oldSum
+      const newFractionOfOthers = 1 - value
```
Comment (MathiasBoi6) [inline packages/synthchron-tauri/src/components/editor/bottomDrawer/postprocessingPanel/PostprocessingPanel.tsx:107]: oldFactionOfOthers, missing an r or intentional?

File: packages/synthchron-tauri/src/components/editor/bottomDrawer/postprocessingPanel/PostprocessingPanel.tsx (hunk 1 above)
Comment (leny-mi) [reply packages/synthchron-tauri/src/components/editor/bottomDrawer/postprocessingPanel/PostprocessingPanel.tsx:25]: Good catch, will fix

File: packages/synthchron-tauri/src/components/editor/bottomDrawer/postprocessingPanel/PostprocessingPanel.tsx (hunk 2 above)
Comment (leny-mi) [reply packages/synthchron-tauri/src/components/editor/bottomDrawer/postprocessingPanel/PostprocessingPanel.tsx:107]: Good catch, will fix

**Review by MathiasBoi6 — APPROVED** [review]
lgtm, but only if PostprocessingPanelOld.tsx, is unused. It seems to be legacy code, so i have not reviewed it.

**Pull request merged**